pyproject.toml

```diff
@@ -35,6 +30,10 @@ pytest = "^7.4.0"
 pre-commit = "^3.3.3"
 requests-mock = "^1.11.0"
 mock = "^5.1.0"
+poethepoet = "^0.21.1"
+
+[tool.poe.tasks]
+install-pyciemss = "pip install git+https://github.com/ciemss/pyciemss.git#c54cff8e301aa53239a9f0e789c8618c3c75569b" 
```

Comment:
Do we really want to tie this to a particular hash?

Reply:
We've decided to start manually updating the hash because we want to verify/test whatever version of PyCIEMSS is going to interacti with Terarium.

Reply:
Ideally though, both MIRA and PyCIEMSS should start publishing their packages to PyPI to make this a bit less nasty.

Reply:
I think we should start requiring PyCIEMSS to freeze the MIRA version as well

Reply:
Any chance we can use tags instead of hashes? That feels like a less obtuse option as hashes are essentially impossible to reason about.

Reply:
We don't have to change to tags today, but would like if we could in the future. This seems like the type of thing that can cause confusion and problems down the line.

Reply:
I agree. I'm going to close this issue for now but create an issue around tagging/cutting releases for PyCIEMSS

Reply:
https://github.com/DARPA-ASKEM/pyciemss-service/issues/35